config/v1/types_console.go

```diff
@@ -26,6 +29,26 @@ type Console struct {
 type ConsoleSpec struct {
 	// +optional
 	Authentication ConsoleAuthentication `json:"authentication"`
+	// customization is used to optionally provide a small set of
+	// customization options to the web console.
+	// +optional
+	Customization ConsoleCustomization `json:"customization,omitempty"`
```

Comment:
I don't see this as customization really. Maybe `route`? Something like

```yaml
spec:
  route:
    hostname: my-hostname
    servingCert: my-secret
```

Reply:
We spoke with @benjaminapetersen that it might be good to follow the patter that we have in our [operator config](https://github.com/openshift/api/blob/master/operator/v1/types_console.go#L31)

Reply:
I don't mean to bikeshed, but this isn't customization imo. It's basic serving info. I wouldn't think to look there to set these values.

(Technically, any config is a customization I guess, but then it starts to lose meaning.)

Reply:
To be honest I like both approaches. Since there already is a `Authentication` field on the `ConsoleSpec` I kinda incline to go with:
```
...
Route   ConsoleRoute   `json:"route,omitempty"`
...
```

@benjaminapetersen thoughts ?

Reply:
I can roll with this.  We have `Customization` on the operator config which contains a `URL`, but its an override to a default.  Agree, this is a different kind of thing. 

I'm good with `Route { URL, ServingCert }` type of struct.

Reply:
https://github.com/openshift/api/blob/master/config/v1/types_infrastructure.go#L58  for an example of documentation verbiage.  

```
URL is a valid URI with scheme(http/https)....
```

Reply:
We want just hostname here, not URL. That matches what Route API, and we don't want the user to set a scheme, port, or path.

Reply:
`hostname` is good, I've been meaning to look up what would be consistent, was thinkging URL isn't quite right.